O/OpenSSH/build_tarballs.jl

```diff
@@ -66,8 +66,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Zlib_jll")
-    Dependency("OpenSSL_jll"; compat="1.1.10")
+    Dependency("Zlib_jll"),
+    Dependency("OpenSSL_jll"),
```

Comment:
Please do not remove compat bounds, update them instead
```suggestion
    Dependency("OpenSSL_jll"; compat="3.0.8"),
```